webviz_ert/views/correlation_view.py

```diff
@@ -11,10 +11,7 @@ def correlation_view(parent: WebvizPluginABC, id_view: str) -> Component:
         className="ert-view-container",
         children=[
             dcc.Graph(
-                id={
-                    "id": id_view,
-                    "type": parent.uuid("graph"),
-                },
+                id=id_view,
```

Comment:
Took a quick look at your PR and it seems the problem in the test happens because two components have the same id, one of them is the graph  `dcc.Graph(id=id_view, ...)` and the other one is the `Div` that contains the graph see lines 9 and 10 in this file  `webviz_ert/views/correlation_view.py`

```python
html.Div(
  id=id_view
```

Dash does not allow two components with the same ids hence the issues you have been seeing in the tests. 

possible fix replace

```
html.Div(
  id=id_view
```
with

```
html.Div(
  id=f"container_{id_view}"
```

Also the function `correlation_view` does not require the `parent` argument anymore.

Reply:
Thanks! The solution seems good. What would you think if we have `view_container` instead of `container` only?
Removing `parent` from `correlation_view` makes the test to fail locally while with it they pass.
Here both fails, so I will investigate...

Reply:
I did not find a reason why the test `test_axes_labels` was failing. When I tried to run again, it passed. I am not sure if this is a flakiness that should be investigate.